src/assets/scss/components/event_settings.scss

```diff
@@ -0,0 +1,20 @@
+$event-icon-palette: (
+    message: #2274d5,
+    inactive: #919191,
+    default: #57a01f
+    ) !default;
+$event-palette: (
+    primary: #57a01f,
+    active: #404040,
+    inactive: #919191,
+    notification: #d10910,
+    background: #fff,
+    background-inactive: #f8f8f8
```

Comment:
tohle je hodně zvláštní paleta - vůbec nebudeš vědet, co ty klíčé znamenají co je klíč background ? Víš jakoby nepříjde mi to ani sémanticky správně.. Čekal jsem paletu přímo barev třeba něco takového : https://tailwindcss.com/docs/colors/#app

Reply:
vzal jsem to na styl Primary color, secondary color atd a podle teto kolekce poskladat celou componentu

Reply:
v poho jako je na to už nějaký best practise, ale vzhledem k rozsahu zadání a požadavků to tady neřešme.. :)